examples/connectivity/README.txt

```diff
@@ -4,3 +4,7 @@ Connectivity Analysis Examples
 
 Examples demonstrating connectivity analysis in sensor and source space.
 
+.. note::
+    Connectivity functionality has moved into the :mod:`mne-connectivity`
+    package. Examples can be found at
+    https://mne.tools/mne-connectivity/stable/auto_examples/index.html.
```

Comment:
... and instead of this being a static URL, would be another intersphinx `:ref:`